logging/zap/examples_test.go

```diff
@@ -74,3 +73,24 @@ func Example_HandlerUsageUnaryPing() {
 		return &pb_testproto.PingResponse{Value: ping.Value}, nil
 	}
 }
+
+func ExampleWithSuppressed() {
+	opts := []grpc_zap.Option{
+		grpc_zap.WithDecider(func(fullMethodName string, err error) bool {
+			// will not log gRPC calls that have the fullMethodName  and no errors
```

Comment:
Seems like comment is incorrect a little bit.

Reply:
updating so this comment is more clear ... we have switched so that by default the result is true (eg. we do log the gRPC call).